projects/alembic/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/alembic/alembic"
+language: c++
+primary_contact: "miller.lucas@gmail.com"
+sanitizers:
+ - address
+ - memory
```

Comment:
Are Alembic and all of its dependencies msan-able (https://google.github.io/oss-fuzz/getting-started/new-project-guide/#sanitizers)?

Reply:
Doesn't look like it. I'll remove the `sanitizers` and `architectures` fields so that OSS-Fuzz can just use default values.